packages/lodestar/src/executionEngine/http.ts

```diff
@@ -97,13 +105,19 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     const method = "engine_newPayloadV1";
     const serializedExecutionPayload = serializeExecutionPayload(executionPayload);
     const {status, latestValidHash, validationError} = await this.rpc
-      .fetch<EngineApiRpcReturnTypes[typeof method], EngineApiRpcParamTypes[typeof method]>({
-        method,
-        params: [serializedExecutionPayload],
-      })
+      .fetchWithRetries<EngineApiRpcReturnTypes[typeof method], EngineApiRpcParamTypes[typeof method]>(
+        {
+          method,
+          params: [serializedExecutionPayload],
+        },
+        {
+          retryAttempts: this.retryAttempts,
+          retryDelay: this.retryDelay,
+        }
+      )
       // If there are errors by EL like connection refused, internal error, they need to be
-      // treated seperate from being INVALID. For now, just pass the error upstream.
-      .catch((e: Error): EngineApiRpcReturnTypes[typeof method] => {
+      // treated separate from being INVALID. For now, just pass the error upstream.
+      .catch(async (e: Error) => {
```

Comment:
I'm not sure if that's allowed, Can a catch callback handle rejects without causing an unhandled promise?